src/components/common/SearchResultsTable/searchResultsTable.tsx

```diff
@@ -322,25 +342,33 @@ const SearchResultsTable = ({
         const aRmp = rmp[searchQueryLabel(convertToProfOnly(a))];
         const bRmp = rmp[searchQueryLabel(convertToProfOnly(b))];
         //drop loading/error rows to bottom
-        if (aRmp.state !== 'done' && bRmp.state !== 'done') {
+        if (
+          (!aRmp || aRmp.state !== 'done') &&
+          (!bRmp || bRmp.state !== 'done')
+        ) {
+          // If both aRmp and bRmp are not done, treat them as equal and return 0
           return 0;
         }
-        if (aRmp.state !== 'done') {
-          return 9999;
+        if (!aRmp || aRmp.state !== 'done') {
+          return -9999;
```

Comment:
I think this should still be positive since we'd want a, being undefined, to come after b.